packages/builder/src/helpers/data/utils.js

```diff
@@ -109,6 +109,36 @@ export function customQueryIconColor(datasource, query) {
   }
 }
 
+export function customQueryText(datasource, query) {
```

Comment:
Great to see the example usage of this in the pr description. Could we add unit tests for this function?

Reply:
Good call actually, it's a perfect candidate. Will add some in.

Reply:
Never mind actually - I forgot that we nuked jest from the builder since it was so horribly broken. This is just something we'll need to remember to add in once we've got a working jest setup again.